examples/player/simid_player.js

```diff
@@ -289,10 +289,10 @@ class SimidPlayer {
       return;
     }
 
-    this.simidIframe_.style.height = incomingMessage.args['height'];
-    this.simidIframe_.style.width = incomingMessage.args['width'];
-    this.simidIframe_.style.left = incomingMessage.args['x_val'];
-    this.simidIframe_.style.top = incomingMessage.args['y_val'];
+    this.simidIframe_.style.height = incomingMessage.args.creativeDimensions['height'];
```

Comment:
Can you add a TODO to make a separate PR that validates the parameters sent it.  It should reject (with an appropriate error message) when the parameters are larger than the player size.

Reply:
I have taken care of it in my PR